howso/details_residuals.amlg

```diff
@@ -30,30 +30,38 @@
 		)
 
 		(declare (assoc
-			local_feature_residuals
+			local_feature_metrics
 				(if (>= model_size 2)
-					(get
-						(call !ExpandResidualValuesToUncertainty  (assoc
-							feature_residuals_map
-								(call !CalculateFeatureResiduals (assoc
-									;only the features from the details
-									features relevant_features
-									case_ids (indices regional_model_cases_map)
-									regional_model_only (true)
-									robust_residuals robust_residuals
-									;use the same hyperparameters that were used to make the original prediction
-									custom_hyperparam_map hyperparam_map
-								))
-						))
-						"residual_map"
+					(call !ExpandResidualValuesToUncertainty  (assoc
+						feature_residuals_map
+							(call !CalculateFeatureResiduals (assoc
+								;only the features from the details
+								features relevant_features
+								case_ids (indices regional_model_cases_map)
+								regional_model_only (true)
+								robust_residuals robust_residuals
+								compute_all_statistics compute_all_statistics
+								store_values store_values
+								;use the same hyperparameters that were used to make the original prediction
+								custom_hyperparam_map hyperparam_map
+							))
+					))
+
+					;can't compute residuals or prediction stats if model is too small
+					(assoc
+						"residual_map" (zip relevant_features 0)
+						"prediction_stats" (zip relevant_features 0)
```

Comment:
prediction stats should be an (assoc) for each feature, right? So `(zip relevant_features 0)` might not make much sense.

Maybe `(zip relevant_features (assoc))` or `(zip relevant_features (null))` would make more sense?